src/sql/src/plan/statement.rs

```diff
@@ -884,4 +884,27 @@ impl<'a> StatementContext<'a> {
     pub fn get_owner_id(&self, id: &ObjectId) -> Option<RoleId> {
         self.catalog.get_owner_id(id)
     }
+
+    /// WARNING! This is meant for internal use only, this should not be used to
```

Comment:
Can this get some words about what is dangerous about it so it's clear even to us when it's appropriate to use?

Reply:
Done